docs/src/usage/equations.md

```diff
@@ -96,6 +96,29 @@ julia> u(0.1)
 0.04999999999996019
 ```
 
+## Eigenvalue Problems
+
+In analogy to linear algebra, many differential equations may be posed as eigenvalue problems. That is, for some differential operator $L$, there are a family of functions $u_i(x)$ such that 
+$$
+L~u_i(x) = \lambda_i u_i(x)
+$$
+where $\lambda_i$ is the $i^{th}$ eigenvalue of the $L$ and has a corresponding *eigenfunction* $u_i(x)$. A classic eigenvalue problem is known as the quantum harmonic oscillator where 
+$$L = -\frac{1}{2}\frac{d^2}{dx^2} + \frac{1}{2} x^2$$
+and one demands that $u(\infty) = u(-\infty) = 0$. Because we expect the solutions to be exponentially suppressed for large $x$, we can approximate this with Dirichlet boundary conditions at a 'reasonably large' $x$ without much difference. 
+
+We can express this in `ApproxFun` as the following:
+```jldoctest
+x = Fun(-8 .. 8)
+L = -𝒟^2/2 + x^2/2
+S = space(x)
+B = Dirichlet(S)
+λ, v = eigs(B, L, 500,tolerance=1E-10) 
+```
+note that boundary conditions must be specified in the call to `eigs`. Plotting the first $20$ eigenfunctions offset vertically by their eigenvalue, we see
+
+![harmonic_eigs](../assets/Harmonic_eigs.pdf)
+
+If the solutions are not relatively constant near the boundary then one should push the boundaries further out. 
```

Comment:
Add a line after this along the lines of:

More generally, `B` can be any zero functional constraint, for example `DefiniteIntegra()`.